packager/export.go

```diff
@@ -0,0 +1,173 @@
+package packager
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/deislabs/cnab-go/bundle"
+	"github.com/deislabs/cnab-go/bundle/loader"
+	"github.com/docker/docker/pkg/archive"
+
+	"github.com/deislabs/duffle/pkg/imagestore"
```

Comment:
This should be imported from `deislabs/cnab-go`.

Reply:
Argh, I thought I had updated that! Sorry!, will amend,